### PR TITLE
Recommend Telegraf in README

### PR DESCRIPTION
**Pull request by remingtonc**
Recommend Telegraf in the README, seem reasonable?
Related to #57

### CODE REVIEW

**Comment by remingtonc** [comment]
ping

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # pipeline-gnmi [![Go Report Card](https://goreportcard.com/badge/cisco-ie/pipeline-gnmi)](https://goreportcard.com/report/cisco-ie/pipeline-gnmi) [![Build Status](https://travis-ci.org/cisco-ie/pipeline-gnmi.svg?branch=master)](https://travis-ci.org/cisco-ie/pipeline-gnmi)
 
-> A streamlined Model-Driven Telemetry collector based on the open-source tool [`pipeline`](https://github.com/cisco/bigmuddy-network-telemetry-pipeline) including enhancements and bug fixes. 
+**NOTE**: Please evaluate usage of the following [Telegraf](https://github.com/influxdata/telegraf) plugins for your use case: [cisco_telemetry_mdt](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/cisco_telemetry_mdt) and [cisco_telemetry_gnmi](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/cisco_telemetry_gnmi). These are more recently developed and allow all the benefits of the Telegraf ecosystem e.g. more output plugins, etc.
```
Comment (skkumaravel) [inline README.md]: I would reword, have the reason upfront. Something like, "For a more recently developed gnmi collector and allow all the benefits of the Telegraf ecosystem, please evaluate usage of the following ..."

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 
-`pipeline-gnmi` is a Model-Driven Telemetry (MDT) collector based on the open-source tool [`pipeline`](https://github.com/cisco/bigmuddy-network-telemetry-pipeline) which has a refreshed codebase improving maintainability, performance, and modern compatibility. It supports MDT from IOS XE, IOS XR, and NX-OS enabling end-to-end Cisco MDT collection for DIY operators.
+> A Model-Driven Telemetry collector based on the open-source tool [`pipeline`](https://github.com/cisco/bigmuddy-network-telemetry-pipeline) including enhancements and bug fixes. 
+
+`pipeline-gnmi` is a Model-Driven Telemetry (MDT) collector based on the open-source tool [`pipeline`](https://github.com/cisco/bigmuddy-network-telemetry-pipeline) which has improved maintainability and compatibility. It supports MDT from IOS XE, IOS XR, and NX-OS enabling end-to-end Cisco MDT collection for DIY operators.
```
Comment (skkumaravel) [inline README.md]: improved maintainability and compatibility than what? awkward sentence.

**Review by skkumaravel — CHANGES_REQUESTED** [review]

**Review by skkumaravel — APPROVED** [review]
LGTM now
